recipes/cai/README.md

```diff
@@ -0,0 +1,23 @@
+# Constitutional AI 
+
+This repo includes the recipe for training 
+
+* https://huggingface.co/HuggingFaceH4/mistral-7b-cai-alpha
+* https://huggingface.co/HuggingFaceH4/mistral-7b-grok-alpha
+
+
+## Full training examples
+
+You will require 8 GPUs (80GB of VRAM) to train the full model.
+```shell
+# Step 1 - SFT
+ACCELERATE_LOG_LEVEL=info accelerate launch --config_file recipes/accelerate_configs/deepspeed_zero3.yaml scripts/run_sft.py recipes/cai/sft/config_full.yaml
+
+# Step 2 - DPO
+ACCELERATE_LOG_LEVEL=info accelerate launch --config_file recipes/accelerate_configs/deepspeed_zero3.yaml scripts/run_dpo.py recipes/cai/dpo/config_full.yaml
+```
+
+
+## Advanced: generating you own dataset
+
+To generate the constitutional AI dataset, see https://github.com/huggingface/llm-swarm/tree/main/examples/hh for detailed instructions if you want build or customize the dataset. 
```

Comment:
I think `hh` is a bit cryptic as the example name in `llm_swarm` - how about we change it to `constitutional-ai` as well?

Reply:
done. https://github.com/huggingface/llm-swarm/tree/main/examples/constitutional-ai